### PR TITLE
lets see if windows likes this...

### PR DESCRIPTION
**Pull request by stefanpenner**

### CODE REVIEW

**Comment by dustinspecker** [comment]
We ran into issues with our Windows CI failing this morning doing a clean `npm install`. It looks like broccoli-caching-writer@2.1.0 is the source of the problems. We also received similar stack traces to [L23 on AppVeyor](https://ci.appveyor.com/project/embercli/broccoli-sourcemap-concat/build/5/job/30cpkfbumyw7t008#L213). `v2.1.0` of broccoli-caching-writer works just fine on Linux, though.

We temporarily fixed it by locking broccoli-caching-writer to `v2.0.4` in our shrinkwrap.

I'm not sure if this is related to what you're running into, but it looks very similar.

**Comment by stefanpenner** [comment]
@dustinspecker you issue should be resolved by: https://github.com/ef4/broccoli-sourcemap-concat/pull/29

I am trying to get windows CI setup and passing so I can release https://github.com/ef4/broccoli-sourcemap-concat/pull/29 with more confidence.

**Comment by stefanpenner** [comment]
@dustinspecker actually if you could try master of this project on your windows machine that would be extremely helpful. At home, It appeared to resolve the issues, but additional validation would be great.

File: test/test.js [hunk 1]
```diff
@@ -374,6 +374,10 @@ describe('concat-without-maps', function() {
   });
 });
 
+function n(string) {
+  return string.replace(/\r\n/, '\n');
```
Comment (kellyselden) [inline test/test.js]: you lost the globalness of the regex

Comment (stefanpenner) [inline test/test.js]: good catch, you are correctly lost the greedy global flag

**Comment by stefanpenner** [comment]
sorry, going to pollute master to get windows CI running...

**Comment by dustinspecker** [comment]
Awesome, thanks.

Master is working perfectly for us.

**Comment by stefanpenner** [comment]
> Master is working perfectly for us.

awesome :) – once CI is green, I will release.

**Comment by kellyselden** [comment]
I don't think this change does anything different. The strings in `n(actualContent), n(expectedContent)` have already been run through `readFileSync` in the lines above.

**Comment by stefanpenner** [comment]
> I don't think this change does anything different. The strings in n(actualContent), n(expectedContent) have already been run through readFileSync in the lines above.

ya, it was out of desperation. :-1:

**Pull request merged**